workspaces/src/network/account.rs

```diff
@@ -40,7 +101,191 @@ impl Contract {
         &self.account.id
     }
 
+    pub fn as_account(&self) -> &Account {
```

Comment:
The contract's account is not necessarily the owner though, since the behavior for an owner can belong to the parent account or the account that created this `Contract`/`Account`